.github/workflows/kdmp-parser.yml

```diff
@@ -13,8 +13,10 @@ jobs:
 
     - name: Install dependencies
       run: |
+        sudo dpkg --add-architecture i386
```

Comment:
Let's forget about Linux 32 bits - I've tried a few things but I am not sure how to tell `CMake` / `FindPython` to look for a different endianess..